Mathlib/SetTheory/Game/Basic.lean

```diff
@@ -101,20 +101,24 @@ If `0 ⧏ x` (less or fuzzy with), then Left can win `x` as the first player. -/
 def LF : Game → Game → Prop :=
   Quotient.lift₂ PGame.LF fun _ _ _ _ hx hy => propext (lf_congr hx hy)
 
-local infixl:50 " ⧏ " => LF
-
 /-- On `Game`, simp-normal inequalities should use as few negations as possible. -/
 @[simp]
-theorem not_le : ∀ {x y : Game}, ¬x ≤ y ↔ y ⧏ x := by
+theorem not_le : ∀ {x y : Game}, ¬x ≤ y ↔ Game.LF y x := by
```

Comment:
I don't understand this change. The statement is the same as before, right?

Reply:
Since this is now in a separate code section, the infix notation is no longer there. I didn't think it was worth it to enable it for two statements.